QGCExternalLibs.pri

```diff
@@ -79,20 +79,20 @@ INCLUDEPATH += libs/qwt
 # Uninstalling from Linux can be done with `sudo make uninstall`.
 #
 XBEE_DEPENDENT_HEADERS += \
-	src/comm/XbeeLinkInterface.h \
-	src/comm/XbeeLink.h \
-	src/comm/HexSpinBox.h \
-	src/ui/XbeeConfigurationWindow.h \
-	src/comm/CallConv.h
+    src/comm/XbeeLinkInterface.h \
```

Comment:
Have you tabs gone wonky? Not sure if that is a tab stop or your tabs are set to 8 instead of 4?